contracts/setup_blending.sh

```diff
@@ -2,24 +2,53 @@
 
 set -euo pipefail
 
-SURI=//Alice
 ROOT_DIR=$(pwd)
 
-TOKENS=1000
+# Actors
+CONTRACTS_ADMIN=//Alice
+DAMIAN=//0
+HANS=//1
+DAMIAN_ACCOUNT=5D34dL5prEUaGNQtPPZ3yN5Y6BnkfXunKXXz6fo7ZJbLwRRH
+HANS_ACCOUNT=5GBNeWRhZc2jXu7D55rBimKYDk8PGk8itRYFTPfC8RJLKG5o
+
+# Token economics
+TOTAL_TOKEN_ISSUANCE=2000
```

Comment:
Would change this to `TOKEN_ISSUANCE_PER_CONTRACT`.

Reply:
sure, changed